src/@chakra-ui/gatsby-plugin/components/Select.ts

```diff
@@ -0,0 +1,68 @@
+import { selectAnatomy } from "@chakra-ui/anatomy"
+import { createMultiStyleConfigHelpers, cssVar } from "@chakra-ui/react"
+import { defineMergeStyles, selectDefaultTheme } from "./components.utils"
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(selectAnatomy.keys)
+
+const $borderBaseWidth = cssVar("border-base-width")
+
+const baseStyleField = defineMergeStyles(selectDefaultTheme.baseStyle?.field, {
```

Comment:
Question.

Noticed that the native select is inheriting all the field styles from the input component. https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/select.ts#L15

Do you think we should merge our Input theme in here instead of the default select?

Something like:
```suggestion
const baseStyleField = defineMergeStyles(OutInputTheme.baseStyle?.field, {
```

The problem I see is that our select & input designs do not share some things, like the focus border. Probably we need to align on that first.

Reply:
> The problem I see is that our select & input designs do not share some things, like the focus border. Probably we need to align on that first.

That was the original concern in pulling the input theme. 

Indeed, the default Chakra theming for Select also inherits styling from the Input, so I would've expected doing the same.

Reply:
All right, lets not do that then yet. We can polish this later.